content/en/blog/2024/sig-cloud-provider-spotlight.md

```diff
@@ -0,0 +1,143 @@
+kind?
+
+**Michael:** I feel that the cloud provider framework is a good place to start as it encodes the
+community’s preferred practices for cloud controller managers, and as such will give a newcomer a
+strong understanding of how and what the managers do. Unfortunately, there is not a consistent
+stream of beginner work on this component, this is due in part to the mature nature of the framework
+and that of the individual providers as well. For folks who are interested in getting more involved,
+having some [Go language](https://go.dev/) knowledge is good, and also having an understanding of
+how at least one cloud API (eg. AWS, Azure, GCP) works is also beneficial. In my personal opinion,
+being a newcomer to SIG cloud provider can be challenging as most of the code around this project
+deals directly with specific cloud provider interactions. My best advice to people wanting to do
+more work on cloud providers is to grow your familiarity with one or two cloud APIs and then look
+for open issues on the controller managers for those clouds, and always communicate with the other
+contributors as much as possible.
+
+## Accomplishments
+
+**Arujjwal:** Can you share about an accomplishment(s) of the SIG that you are proud of?
+
+**Michael:** Since I joined the SIG, more than a year ago, we have made great progress in advancing
+the extraction and migration subproject. We have moved from an alpha status on the defining
+[KEP](https://github.com/kubernetes/enhancements/blob/master/keps/README.md) to a beta status and
+are inching ever closer to removing the old provider code from the Kubernetes source tree. I've been
+really proud to see the active engagement from our community members and to see the progress we have
+made towards extraction, I have a feeling that within the next few releases, we will see the final
+removal of the in-tree cloud controllers and the completion of the subproject.
+
+## Advice for new contributors
+
+**Arujjwal:** Is there any suggestion or advice for new contributors on how they can start at SIG
+Cloud Provider?
+
+**Michael:** This is a tricky question in my opinion. SIG cloud provider is focused on the code
+pieces that integrate between Kubernetes and an underlying infrastructure. It is very common, but
+not necessary, for members of the SIG to be representing a cloud provider in an official capacity. I
+recommend that anyone interested in this part of Kubernetes should come to an SIG meeting to see how
+we operate and also to study the cloud provider framework project. We have some interesting ideas
+for future work, such as a common testing framework, that will cut across all cloud providers and
+will be a great opportunity for anyone looking to expand their Kubernetes involvement.
+
+**Arujjwal:** Are there any specific skills you're looking for that we should highlight? To give you
+an example from our own SIG Contribex: if you're an expert in [Hugo](https://gohugo.io/), we can
```

Comment:
```suggestion
an example from SIG Contributor Experience: If you're an expert in [Hugo](https://gohugo.io/), we can
```

Reply:
“SIG ContribEx” if that's what was said at the time.

Reply:
Fair enough. I'd still add an editor's note, like as follows or similar, to make it easier for the reader who isn't familiar with the SIGs:
```suggestion
an example from our own SIG Contribex _[ed. - SIG Contributor Experience]_: If you're an expert in [Hugo](https://gohugo.io/), we can
```

Reply:
I think we can go with the suggestion (I'll confirm with Arujjwal). I slightly changed this since the original had only "our group" which was not something readers would know about. I'll come back to this one later.

Reply:
Another approach: hyperlink to SIG ContribEx. That's my recommendation.

Reply:
Tha'ts even better. Will take that one.